pkg/storage/swift/swift_test.go

```diff
@@ -335,6 +335,51 @@ func TestSwiftRemoveStorageWithContent(t *testing.T) {
 	th.AssertEquals(t, true, containerDeleted)
 }
 
+func TestSwiftRemoveStorageWithContentFailure(t *testing.T) {
+	th.SetupHTTP()
+	defer th.TeardownHTTP()
+	handleAuthentication(t, "container")
+
+	var containerContentListed bool
+	var containerContentDeleted bool
+	var containerDeleted bool
+	th.Mux.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
+		switch r.Method {
+		case http.MethodGet:
+			objects := []string{"obj0", "obj1"}
+			if containerContentListed {
+				objects = []string{}
+			}
+			containerContentListed = true
+			w.WriteHeader(http.StatusOK)
+			_ = json.NewEncoder(w).Encode(objects)
+		case http.MethodPost:
+			if containerContentDeleted {
+				w.WriteHeader(http.StatusNotFound)
+				return
+			}
+			containerContentDeleted = false
+			w.WriteHeader(http.StatusOK)
+			respBody := `{"Number Not Found": 0, "Response Status": "200 OK", "Errors": [["obj0", "Internal Error"]], "Number Deleted": 1, "Response Body": ""}`
+			_, _ = w.Write([]byte(respBody))
+		case http.MethodDelete:
+			w.WriteHeader(http.StatusNoContent)
+			containerDeleted = true
+		}
+	})
+
+	d, installConfig := mockConfig(true, th.Endpoint()+"v3", MockUPISecretNamespaceLister{})
```

Comment:
`mockConfig`'s signature was changed with https://github.com/openshift/cluster-image-registry-operator/pull/596/files?file-filters%5B%5D=.go#diff-6cddc54abe4f6fba085c6fc5b982b9d9 and this call needs to be updated.